docs/root/configuration/cluster_manager/cds.rst

```diff
@@ -24,6 +24,7 @@ CDS has a statistics tree rooted at *cluster_manager.cds.* with the following st
   :widths: 1, 1, 2
 
   config_reload, Counter, Total API fetches that resulted in a config reload due to a different config
+  init_fetch_timeout, Counter, Total :ref:`initial fetch timeouts <envoy_api_field_core.ConfigSource.initial_fetch_timeout>`
```

Comment:
Thanks, this is a really useful change. However, I think with all of our xDS, it's going to be hard to keep each doc spot up to date. WDYT about maybe have a central doc for "common subscription stats" which we can link to from all the various xDS documentation? cc @htuch

Reply:
That makes sense. I can add a central doc for "common subscription stats" - I can do a follow-up if it is easier to review the just doc part or if you think it is better to do it as part of this, I am fine with that too. LMK what you think?

Reply:
I think it's fine to do it as part of this change as it's a pretty small change. Thank you!

Reply:
Yep, good idea!

Reply:
@mattklein123 added the subscription_stats page and linked every where else. PTAL.